zenith/src/main.rs

```diff
@@ -63,6 +66,15 @@ fn main() -> Result<()> {
                 .subcommand(SubCommand::with_name("stop").arg(name_arg.clone()))
                 .subcommand(SubCommand::with_name("destroy").arg(name_arg.clone())),
         )
+        .subcommand(
```

Comment:
We'll need some more documentation for this. Not sure where, we don't have any user-documentation at all yet :-). I guess some comments here or down in `handle_origin` will do for now.

What format is the url? Looking at the code, seems it's a postgres connections string. That can be a URL, but it can also be key-value list separated by spaces, something like `dbname=postgres user=heikki`.

Reply:
Intention was to have connection uri here: https://github.com/zenithdb/rfcs/blob/push-pull/push-pull.md#origin-management as git mostly uses uri-like addresses. Doesn't hurt to support usual postgres connection string though

Reply:
Yep that was my intention too, I used the postgres crate to do the validation (but that also happens to support connection strings). I can drop this validation, try to write my own regex thing for URIs, or maybe see if I can get access to the URI parser in `postgres` crate